src/components/alert/alert.tsx

```diff
@@ -413,4 +429,12 @@ export class Alert implements OpenCloseComponent, LocalizedComponent, LoadableCo
     window.clearTimeout(this.queueTimeout);
     this.queueTimeout = window.setTimeout(() => (this.queued = false), 300);
   }
+
+  handleActionsEndSlotChange = (event: Event): void => {
```

Comment:
You can remove this function in favor of this dom utility.

https://github.com/Esri/calcite-components/blob/3eb037e627832c34334969d0dc0cf5dde3ce42bc/src/utils/dom.ts#L315-L317